reference/5.1/Microsoft.PowerShell.Core/Invoke-History.md

```diff
@@ -21,62 +21,76 @@ Invoke-History [[-Id] <String>] [-WhatIf] [-Confirm] [<CommonParameters>]
+This example runs the last, or most recent, command in the session history. You can abbreviate this
+command as `r`, the alias for `Invoke-History`.
 
-### Example 2: Run the command that has a specified ID
-```
-PS C:\> Invoke-History -Id 132
+```powershell
+Invoke-History
 ```
 
-This command runs the command in the session history with ID 132.
-Because the name of the *Id* parameter is optional, you can abbreviate this command as the following: `Invoke-History 132`, `ihy 132`, or `r 132`.
+### Example 2: Run the command that has a specified ID
+
+This example runs the command in the session history with **Id** 132. Because the name of the **Id**
+parameter is optional, you can abbreviate this command as the following: `Invoke-History 132`,
+`ihy 132`, or `r 132`.
 
-### Example 3: Run the most recent command by using the command text
+```powershell
+Invoke-History -Id 132
 ```
-PS C:\> Invoke-History -ID get-pr
+
+### Example 3: Run the most recent command by using the command text
+
+This example runs the most recent `Get-Process` command in the session history. When you type
+characters for the **Id** parameter, `Invoke-History` runs the first command that it finds that
+matches the pattern, starting with the most recent commands.
+
+```powershell
+Invoke-History -Id get-pr
 ```
 
-This command runs the most recent **Get-Process** command in the session history.
-When you type characters for the *Id* parameter, **Invoke-History** runs the first command that it finds that matches the pattern, starting with the most recent commands.
+> [!NOTE]
+> Pattern matching is case-insensitive, but the pattern matches the beginning of the line.
 
 ### Example 4: Run a sequence of commands from the history
-```
-PS C:\> 16..24 | ForEach {Invoke-History -Id $_ }
-```
 
-This command runs commands 16 through 24.
-Because you can list only one ID value, the command uses the ForEach-Object cmdlet to run the **Invoke-History** command one time for each ID value.
+This example runs commands 16 through 24. Because you can list only one **Id** value, the command
+uses the `ForEach-Object` cmdlet to run the `Invoke-History` command one time for each **Id** value.
 
-### Example 5
-```
-PS C:\> Get-History -Id 255 -Count 7 | ForEach {Invoke-History -Id $_.Id}
+```powershell
+16..24 | ForEach {Invoke-History -Id $_ }
 ```
 
-This command runs the 7 commands in the history that end with command 255 (typically 249 through 255).
-It uses the Get-History cmdlet to retrieve the commands.
-Because you can list only one ID value, the command uses the ForEach-Object cmdlet to run the **Invoke-History** command once for each ID value.
+### Example 5
+
+This example runs the seven commands in the history that end with command 255 (249 through 255). It
+uses the `Get-History` cmdlet to retrieve the commands. Because you can list only one **Id** value,
+the command uses the `ForEach-Object` cmdlet to run the `Invoke-History` command once for each
+**ID** value.
```

Comment:
It's `Id` property in all other mentions.

Reply:
Fixed